pkg/apis/eventing/v1alpha1/eventpolicy_lifecycle.go

```diff
@@ -20,10 +20,12 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var eventPolicyCondSet = apis.NewLivingConditionSet()
+var eventPolicyCondSet = apis.NewLivingConditionSet(EventPolicyConditionAuthnEnabled, EventPolicyConditionSubjectsResolved)
 
 const (
-	EventPolicyConditionReady = apis.ConditionReady
+	EventPolicyConditionReady                               = apis.ConditionReady
+	EventPolicyConditionAuthnEnabled     apis.ConditionType = "AuthenticationEnabled"
+	EventPolicyConditionSubjectsResolved apis.ConditionType = "SubjectsResolved"
```

Comment:
It would be nice to have some Marker methods for these conditions. Check on the trigger how it sets its conditions: https://github.com/knative/eventing/blob/0f71292d9887001caef68ce9fdd886596d4df276/pkg/apis/eventing/v1/trigger_lifecycle.go#L209-L223

Reply:
@creydr no strong opinions but it seems like a bit of indirection as the markers are simply setters methods.

Reply:
Correct, they are simple setter methods. They can help you later then e.g. in https://github.com/knative/eventing/pull/8024/files#r1654643259

Reply:
I added these marker methods in the new commit.